iOS/PostEditor/PostTextEditingView.swift

```diff
@@ -60,8 +60,8 @@ struct PostTextEditingView: View {
                 }
             }
         }
-        .onChange(of: titleIsFirstResponder, perform: { _ in
-            self.bodyIsFirstResponder.toggle()
+        .onChange(of: titleIsFirstResponder, perform: { value in
+            self.bodyIsFirstResponder = !value
```

Comment:
Making explicit that `bodyIsFirstResponder` should be the negation of `titleIsFirstResponder`.